packages/ilib-loctool-webos-javascript/test/testfiles/js/t5.js

```diff
@@ -0,0 +1,6 @@
+//
+[GAME]: () => $L("Enjoy various games"), // tips_15
+//
+[HOME_HUB]: () => $L("Control smart home IoT devices"), // tips_40
```

Comment:
One of these three strings should have an URL in it to make sure you get the right string extracted!

```suggestion
[HOME_HUB]: () => $L("Control smart home IoT devices. See https://lge.com/smarthome for details."), // tips_40
```

Reply:
Yes, The test string has been updated.